src/routes/v2/authenticated/sites.ts

```diff
@@ -63,7 +63,7 @@ export class SitesRouter {
     { userWithSiteSessionData: UserWithSiteSessionData }
   > = async (req, res) => {
     const { userWithSiteSessionData } = res.locals
-    const lastUpdated = await this.sitesService.getLastUpdated(
+    const lastUpdated = this.sitesService.getLastUpdated(
```

Comment:
hey sanity check here, is this intended?

Reply:
Yes! The function no longer returns a promise